doc/developer-documentation/migration-to-new-version.md

```diff
@@ -0,0 +1,127 @@
+# Migration guide from Etherlime v1.2 to v2.0
+
```

Comment:
Lets add the following:

"TLDR: 
1. In your project run `npm i etherlime-lib`. 
2. Wherever you have `require('etherlime')` you now need to change to `require('etherlime-lib'). 
3. You can safely uninstall your project version of etherlime `npm uninstall etherlime`.
Note: Do reinstall the global CLI version of etherlime - `npm uninstall -g etherlime`, `npm install -g etherlime`"